pkg/api/helpers.go

```diff
@@ -449,9 +449,9 @@ func GetAffinityFromPodAnnotations(annotations map[string]string) (*Affinity, er
 	return nil, nil
 }
 
-// GetTolerationsFromPodAnnotations gets the json serialized tolerations data from Pod.Annotations
+// GetTolerationsFromAnnotations gets the json serialized tolerations data from Annotations
 // and converts it to the []Toleration type in api.
-func GetTolerationsFromPodAnnotations(annotations map[string]string) ([]Toleration, error) {
+func GetTolerationsFromAnnotations(annotations map[string]string) ([]Toleration, error) {
```

Comment:
It's possible that downstream code (OpenShift) could use it. I'm comfortable with the rename, however, and if OpenShift does use it, we'll just fix it in a rebase.